release_template.md

```diff
@@ -0,0 +1,31 @@
+
+---------------------------------------------------------------------
+Release {TagName}
+---------------------------------------------------------------------
+New features
+--------
+- #1767: Add full account history csv export
```

Comment:
how was this content created? Or what is the intention for using this template in a release flow?

Reply:
I think it's not a template but auto-generated release notes, and it will be used as the final release notes when doing a release, and we can polish it as we like before the final release. Perhaps it's not the best approach though. @sschiessl-bcp what do you think?

Reply:
This template is prepared for the automatic release of Github action. I think that each release should have corresponding instructions. The only placeholders currently supported are tags and hashes of binary files. Any contributor can maintain this template. The github action will use the content of this file as the release note when the version tag of